src/node_options.cc

```diff
@@ -324,6 +324,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "experimental WASI support",
             &EnvironmentOptions::experimental_wasi,
             kAllowedInEnvironment);
+  AddOption("[harmony_top_level_await]",
+            "",
+            &EnvironmentOptions::harmony_top_level_await);
+  Implies("--harmony-top-level-await", "[harmony_top_level_await]");
```

Comment:
Since the entire modules implementation is considered experimental, I'd strongly suggest shipping top-level await by default without requiring a flag.

Reply:
Can't; flags invalidate the v8 snapshot system

Reply:
Couldn't we snapshot with TLA enabled?

Reply:
@targos i think we would have to patch v8 to set the flag's default value to enabled, since it has its own startup snapshot baked in.